packages/contract-method-name/index.test.js

```diff
@@ -0,0 +1,45 @@
+import {expect, describe, beforeEach} from '@jest/globals'
+import {getEthContractName, getCFXContractName} from './index.js'
+let contractAddress = ''
+let transactionData = ''
+const fakeContractAddress = 'iamfakeaddress'
+const fakeTransactionData = 'iamfaketx'
+describe('contract-method-name', function () {
```

Comment:
don't rely on external resources like external fullnode or scan api in tests
- let user pass in the rpc provider and mock them in tests
- use things like jest mock, nock or api provided by the fetch library, like [hooks in ky](https://github.com/sindresorhus/ky#hooksbeforerequest) to mock the request